setup.py

```diff
@@ -2,6 +2,8 @@
 
 from __future__ import absolute_import, division, print_function
 
+import setuptools
```

Comment:
I can only see that happening if you use a environment that somehow import setuptools before running the `setup.py`. Note that distutils documentation do not mentioned either option.

If I use python 2.7.9 from ubuntu, this is what I see:

```
/usr/lib/python2.7/distutils/dist.py:267: UserWarning: Unknown distribution option: 'entry_points'
  warnings.warn(msg)
/usr/lib/python2.7/distutils/dist.py:267: UserWarning: Unknown distribution option: 'install_requires'
  warnings.warn(msg)
running config
```

Those warnings disappear after importing setuptools.